docs/docs/tutorial.md

```diff
@@ -221,7 +222,7 @@ Note that we have passed some data from the parent `CommentList` component to th
 
 Markdown is a simple way to format your text inline. For example, surrounding text with asterisks will make it emphasized.
 
-First, add the third-party library **marked** to your application. This is a JavaScript library which takes Markdown text and converts it to raw HTML. This requires a script tag in your head (which we have already included in the React playground):
+In this tutorial we use a third-party library **marked** which takes Markdown text and converts it to raw HTML. This requires a script tag in your head (which we have already included in the React playground):
```

Comment:
A couple changes. First, let's change the 2nd sentence here.

> This requires a script tag in your head (which we have already included in the React playground):

Let's just make that:

> We already included this library with the original markup for the page, so we can just start using it.

And then let's get rid of this whole HTML code block below. It's just noise since the script is already there.